vision_agent_tools/models/florencev2.py

```diff
@@ -98,8 +113,20 @@ def __call__(
         else:
             text_input = task + prompt
 
-        image = image.convert("RGB")
+        # Either video or image should be provided
+        if image is None and video is None:
+            raise ValueError("Either 'image' or 'video' must be provided.")
+        if image is not None and video is not None:
+            raise ValueError("Only one of 'image' or 'video' can be provided.")
+
+        if image is not None:
+            image = self._process_image("RGB")
```

Comment:
Looks like the tests are failing due to this call

`_process_image` is defined as 
```python
    def _process_image(self, image: Image.Image) -> Image.Image:
        return image.convert("RGB")
```

so it is executing `"RGB".convert("RGB")` thus the error

```
E       AttributeError: 'str' object has no attribute 'convert'
```